handbook/usage.md

```diff
@@ -66,6 +66,17 @@ If you use Google Chrome, you can add a Sourcegraph handbook search shortcut to
    - URL with %s in place of query: https://about.sourcegraph.com/search?q=%s
 1. Press the **Add** button.
 
+### Firefox 
+
+If you use Firefox, you can add a Sourcegraph handbook search shortcut to the address bar
+
+#### Setup 
+
+1. Right click the search bar on https://about.sourcegraph.com/handbook 
+2. Choose "Add a keyword for this Search"
+3. Enter a keyword that you would like to use (e.g. `sg`)
```

Comment:
This should recommend using hb as the shortcut for consistency with the Chrome instructions. Not a big deal but just potential for confusion if it’s inconsistent.

Reply:
Updated 👍